gregBorchers/Homework2GregBorchers-probs1-4NoCards.js

```diff
@@ -0,0 +1,359 @@
+// PART 1 OF 2, for January 19 JSE Homework. /*Due Monday, 1/19
+/*
+Each problem below lists an estimate of its relative difficulty and how much
+time it requires as a percentage of the whole set.  The total time estimate for
+this homework set is around 10-15 hours.
+
+Please also read the [learning objectives](objectives.md) for this week.
+
+
+
+**1) _(Easy, 10% of total time)_
+
+Revisit your solution to homework #1, problem 5, which expresses an improper
+fraction as a proper one, and turn your solution into a function.
+
+**a)** Write a function `fractionString(n,d)` which takes 2 parameters (n,d) and
+returns a string.  For example,  `fractionString(7,4)` should return "1 3/4",
+and `fractionString(3,3)` should (for now) return "1 0/3".  As before, assume
+that _n_ and _d_ are both positive integers.
+
+Note that _returning_ a string is not the same as _printing_ a string.  You may
+use `console.log()` for debugging, but your function should have the correct
+string as its return value. You can also print your return value with an
+expression like `console.log(fractionString(7,4))`. 
+*/
+
+var n =3;
+var d = 2;
+
+var fractionString = function(n,d) {
+  var wholeNumber = 0;
+  var numerator = 0;
+  var properFraction = "";
+
+  wholeNumber = Math.trunc( n / d );
+  numerator = n % d;
+
+  properFraction = wholeNumber + " " + numerator + "/" + d;
+
+  return properFraction;
+
+}
+
+var properFractionString = fractionString(n,d);
+console.log(properFractionString);
```

Comment:
1) Looks good! You can eliminate a couple of extra lines in there by getting rid of the initial declarations of variables and moving the `var` to the equation, like `var numerator = n % d;`